internal/server/http/handler/v1/treecluster/handler.go

```diff
@@ -33,23 +35,8 @@ func GetAllTreeClusters(svc service.TreeClusterService) fiber.Handler {
 	return func(c *fiber.Ctx) error {
 		ctx := c.Context()
 
-		pageParam, err := strconv.Atoi(c.Query("page", "1"))
-		if err != nil || pageParam < 1 {
-			err := service.NewError(service.BadRequest, "invalid page format")
-			return errorhandler.HandleError(err)
-		}
-		page := int32(pageParam)
-
-		limit := int32(-1)
-		limitParam := c.Query("limit", "-1")
-		if limitParam != "" {
-			limitParam, err := strconv.Atoi(limitParam)
-			if err != nil || (limitParam != -1 && limitParam <= 0) {
-				err := service.NewError(service.BadRequest, "invalid limit format")
-				return errorhandler.HandleError(err)
-			}
-			limit = int32(limitParam)
-		}
+		page := c.Locals(middleware.Page).(int32)
+        limit := c.Locals(middleware.Limit).(int32)
 
 		domainData, totalCount, err := svc.GetAll(ctx, page, limit)
```

Comment:
The functions, in this case `GetAll`, does not need to take `page` and `limit` as parameters. These attributes are set in the context